src/alchemlyb/parsing/namd.py

```diff
@@ -25,72 +86,147 @@ def extract_u_nk(fep_file, T):
     u_nk : DataFrame
         Potential energy for each alchemical state (k) for each frame (n).
 
+    Note
+    ----
+    If the number of forward and backward samples in a given window are different,
+    the extra sample(s) will be discarded. This is typically zero or one sample.
 
     .. versionchanged:: 0.5.0
         The :mod:`scipy.constants` is used for parsers instead of
         the constants used by the corresponding MD engine.
 
+        Support for Interleaved Double-Wide Sampling files added. 
```

Comment:
Need new version here:
```suggestion
    .. versionchanged:: 0.6.0
        Support for Interleaved Double-Wide Sampling files added.
```